fissa/extraction.py

```diff
@@ -275,6 +275,169 @@ def extracttraces(data, masks):
         return out
 
 
+class DataHandlerTifffileLazy(DataHandlerAbstract):
+    """
+    Extract data from TIFF images using tifffile, with lazy loading.
+    """
+    @staticmethod
+    def image2array(image):
+        """
+        Load a TIFF image from disk.
+
+        Parameters
+        ----------
+        image : str or array_like
+            Either a path to a TIFF file, or array_like data.
```

Comment:
The current function can only take in a TiffFile object, no arrays.

Example:
```
import tifffile
import numpy as np
image = np.random.rand(10, 10,10)
tifffile.TiffFile(image)
```
will give an error. 
(Specifically, TiffFile can take as a first argument 'a file name, seekable binary stream, or FileHandle')